azure-kusto-data/source/client.js

```diff
@@ -31,33 +31,33 @@ module.exports = class KustoClient {
         };
     }
 
-    execute(db, query, callback, properties) {
+    async execute(db, query, properties) {
         query = query.trim();
         if (query.startsWith(".")) {
-            return this.executeMgmt(db, query, callback, properties);
+            return this.executeMgmt(db, query, properties);
         }
 
-        return this.executeQuery(db, query, callback, properties);
+        return this.executeQuery(db, query, properties);
     }
 
-    executeQuery(db, query, callback, properties) {
-        return this._execute(this.endpoints.query, db, query, null, callback, properties);
+    async executeQuery(db, query, properties) {
+        return this._execute(this.endpoints.query, db, query, null, properties);
     }
 
-    executeMgmt(db, query, callback, properties) {
-        return this._execute(this.endpoints.mgmt, db, query, null, callback, properties);
+    async executeMgmt(db, query, properties) {
+        return this._execute(this.endpoints.mgmt, db, query, null, properties);
     }
 
-    executeStreamingIngest(db, table, stream, streamFormat, callback, properties, mappingName) {
```

Comment:
Why properties was removed?

Reply:
We don't use it, streamingIngestClient call this method with properties = null